src/modules/inventory/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ project(Inventory)
 include(../../cmake/CommonSettings.cmake)
 set_common_settings()
 
+if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /WX-")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} /WX-")
+    add_compile_options(/w)
+endif()
```

Comment:
I wonder if it could work like this...

```
if (CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
    target_compile_options(Inventory PRIVATE /WX- /w)
endif()
```

(should probably come after line 23 (because of the `add_library`...)

If it still doesn't work we could try to find the targets that need these flags, instead of applying them to the entire project.

Reply:
Done!